tests/conn_can/Makefile

```diff
@@ -10,8 +10,10 @@ USEMODULE += conn_can_isotp_multi
 USEMODULE += can_pm
 USEMODULE += can_trx
 
-FEATURES_REQUIRED += periph_can
-FEATURES_REQUIRED += periph_gpio_irq
+ifneq ($(BOARD), native)
```

Comment:
this needs to go into `tests/conn_can/Makefile.dep`.
Otherwise, it won't be considered by e.g., `make info-boards-supported`.

Reply:
The filename should be `tests/conn_can/Makefile.board.dep`.

https://github.com/RIOT-OS/RIOT/blob/02c16479cf45267e30c7eeef53719e5a61782279/Makefile.dep#L5-L6

Reply:
Sorry for the rookie mistakes. I think I made the requested changes. Can anyone confirm before I Resolve the conversation?

Reply:
This application will work for boards other than native that don't have the periph_can driver, using eg. the mcp2515 driver. So I think these `FEATURES_REQUIRED` lines can be removed.